### ISSUE TITLE
Accessibility Audit: The language of this page is not set.

### ISSUE THREAD

**Issue by zepumph** [phetsims/joist#492]
After auditing a few version of phet sims (and different locales), it seems like we aren't setting the `lang` attribute for the main `<html>`:

![image](https://user-images.githubusercontent.com/6856943/39898334-bc2c29a2-5462-11e8-9c28-8f4305cc449f.png)

Audited with https://tenon.io/testNow.php?url=https://phet.colorado.edu/sims/html/faradays-law/latest/faradays-law_be.html

This may be a bit of work to do, because it should be filled in based on the build. Not sure what the `allHTML` files should be set to. @terracoda how important is this in your mind?

**Comment by zepumph** [comment]
Tagging @terracoda for discussion. This was brought up in an a11y meeting today.

**Comment by zepumph** [comment]
I wonder if it is possible to have the language set dynamically, since we can change locale based on a query parameter.

**Comment by zepumph** [comment]
Tagging @jessegreenberg too.

**Comment by jessegreenberg** [comment]
It was also brought up that we have some unicode marks that surrounds our translatable strings.
Sometimes VoiceOver says "left to right embedding marks" when it encounters one, depending on user settings. I actually  haven't heard that in a while, maybe default settings were adjusted.

**Comment by terracoda** [comment]
@zepumph and @jessegreenberg, I also have not heard, "left to right embedding" in a while, so maybe that issue has been fixed in some natural way.

**Comment by terracoda** [comment]
It is my understanding that the `lang` attribute is very important for accessibility, see this short article by Léonie Watson, [Using the HTML `lang` attribute](https://developer.paciellogroup.com/blog/2016/06/using-the-html-lang-attribute/).

In our case, it may not be necessary to put the lang attribute on the parent `html `element as it is my understanding that we only have text-based content in our accessible PDOM. 
Would it be easier to assign the language dynamically on the accessibility `div`?

**Comment by terracoda** [comment]
Un-assigning myself from this issue. Please re-assign when needed.

**Comment by jessegreenberg** [comment]
The lang attribute was added in https://github.com/phetsims/scenery/issues/1039, closing. We also remove embedding marks from description strings to prevent "left to right embedding" from being read. Closing.
